### ISSUE TITLE
Deter users from emailing a password in response to a password reset email

### ISSUE THREAD

**Issue by RichardTaylor** [mysociety/alaveteli#7431]
We've had a couple of cases of users replying to emails eg. the 

>Please click on the link below to confirm your email address. Then you can change your password on WhatDoTheyKnow

and the 

>Change your password on WhatDoTheyKnow

emails with a password rather than following the links. 

I suspect these are rare enough edge cases that no action might be appropriate. 

I doubt we'll want to add clutter for the vast majority of users, just to attempt to deal with edge cases. 

Perhaps there is action we can take, or discussion to be had, though, hence this ticket.

**Comment by HelenWDTK** [comment]
This issue has been automatically closed due to a lack of discussion or resolution for over 12 months.
Should we decide to revisit this issue in the future, it can be reopened.
